integration-tests/build.gradle

```diff
@@ -374,6 +374,22 @@ task runContainerTests(type: Test) {
     }
 }
 
+task runContainerSAFProviderTests(type: Test) {
```

Comment:
isn't saf provider efectively a dummy in the GH actions?

Reply:
for the token generation yes, but there's differing paths in logout between z/OSMF and SAF, which is the scenario that I wanted to test as a security measure